docs/search/query-assist.md

```diff
@@ -0,0 +1,87 @@
+2. **Leverage syntax and schema suggestions**. As you type, watch for real-time suggestions for fields and operators. Select options using your arrow keys and press `Tab` to auto-complete.
+3. **Take advantage of predictions**. Use partial and full query predictions to build queries faster and more accurately.
+4. **Review and correct errors**. Correct flagged errors before running your query.
+
+## Features
+
+## Token-by-token prediction and auto-complete
+
+Get real-time suggestions for query completion as you type, with token-by-token predictions helping to quickly finish your queries.
+
+As you start typing a query, such as `_sourceCategory=`, the system provides **token-by-token predictions** and auto-completes your query with relevant metadata options. Simply use the `Tab` key to accept the suggestion. For example, after typing `status=`, you will be prompted with likely values such as `success`, `error`, or other status codes based on your previous queries or log data.
+
+## Schema discovery and field suggestions
+
+Automatically receive suggestions for relevant fields in structured data like JSON logs, making field discovery much easier.
+
+For structured logs like JSON, the system automatically suggests relevant fields such as `userID`, `eventType`, or `timestamp` as you type. This eliminates the need for manual inspection of logs, making it easier to filter and aggregate data.
+
+## Next operator and full query predictions
+
+The system intelligently predicts the next operator or offers full query suggestions based on your input, reducing manual effort.
+
+When you type a query such as `source=logs | where status="error"`, the system predicts the **next operator** and offers suggestions like `count by employeeID` or other common fields. This minimizes manual input and ensures your queries are accurate and aligned with standard query patterns.
+
+## Real-time error highlighting and feedback
+
+Errors are flagged as you type, with immediate suggestions for corrections, ensuring a seamless query-writing experience.
+
+As you write queries, the system highlights any syntax errors in real-time. For example, if you forget to close parentheses or misspell an operator, the system will flag it and provide suggestions to fix the issue immediately, preventing unnecessary troubleshooting.
+
+## Contextual auto-complete and field discovery
+
+Suggestions are ranked based on your organization’s common queries, making query completion smarter and more relevant.
+
+The system ranks suggestions based on common queries from your organization or your own previous queries. This ensures that your auto-complete options are not only relevant but also contextually accurate, speeding up the query-writing process.
+
+
+### Need more help?
+
+Try [Sumo Logic Copilot](/docs/search/copilot), our AI-powered assistant that helps you write and execute natural language log search queries.
```

Comment:
Why is this captured here ?

Reply:
I wasn't sure if you wanted to cross-promote copilot & query assist. I'll remove it.